app/routes.js

```diff
@@ -1,7 +1,7 @@
-// const controller = require('./controllers/controller');
+const userController = require('./controllers/user'),
+  userValidator = require('./middlewares/userValidator'),
+  passwordEncrypt = require('./middlewares/passwordEncrypt');
 
 exports.init = app => {
-  // app.get('/endpoint/get/path', [], controller.methodGET);
-  // app.put('/endpoint/put/path', [], controller.methodPUT);
-  // app.post('/endpoint/post/path', [], controller.methodPOST);
+  app.post('/users', userValidator, passwordEncrypt, userController.save);
```

Comment:
we usuallu add middlewares as a list, like [userValidator, passwordEncrypt]. I did not know it can work the way you did it